Automattic/wp-calypso
# UpsellNudge: Replace instances of Banner with UpsellNudge

#### Changes proposed in this Pull Request

* Swap out instances of `Banner` with `UpsellNudge` as per #38778

| Visuals | | 
| --- | --- |
| <img width="1104" alt="Screen Shot 2020-04-02 at 2 16 23 PM" src="https://user-images.githubusercontent.com/2124984/78284725-d520f700-74ed-11ea-80c5-73039512f802.png"> | <img width="1106" alt="Screen Shot 2020-04-02 at 2 40 21 PM" src="https://user-images.githubusercontent.com/2124984/78289565-609b8780-74f0-11ea-9aea-4b3d8b8b01cf.png"> |
| <img width="1092" alt="Screen Shot 2020-04-02 at 2 47 01 PM" src="https://user-images.githubusercontent.com/2124984/78289943-eddedc00-74f0-11ea-846a-259152e66c06.png"> | <img width="783" alt="Screen Shot 2020-04-02 at 2 52 09 PM" src="https://user-images.githubusercontent.com/2124984/78290506-d2280580-74f1-11ea-929e-5cabf7e6b0dc.png"> | 
| <img width="735" alt="Screen Shot 2020-04-02 at 3 26 56 PM" src="https://user-images.githubusercontent.com/2124984/78293351-7318bf80-74f6-11ea-9c34-391700ed27bf.png"> | 


#### Testing instructions

Switch to this PR and test the upsells in the following scenarios:

* On a site with the Blogger plan, go to `/domains/add/transfer/{site}` and attempt to add a `.com` domain. An upsell nudge should appear. (It has no associated Tracks events.)
* On a site with the free plan, go to `/activity-log/{site}` and there should be an upsell at the top of the screen
* On a site with the free plan, go to `/hosting-config/{site}` and note the nudge
* On a site with the free plan, go to Plugins -> Click on any individual plugin and note the upsell
* On a site with the free plan, go to Settings -> General -> Scroll to Footer credit and note the upsell (This has no associated Tracks events.)
